tests/port_dpb.py

```diff
@@ -242,8 +242,10 @@ def breakin(self, dvs, port_names):
 
         for cp in child_ports:
             assert(cp.exists_in_config_db() == False)
+        time.sleep(1)
```

Comment:
Why do we need the sleep here?

Reply:
I've saw this failing sometime and the reason i thought was this https://github.com/sonic-net/sonic-swss/pull/2431#issuecomment-1230499090. The idea is to stabilize

Reply:
In https://github.com/sonic-net/sonic-swss/pull/2431#issuecomment-1230499090, you mentioned that it did not help...

Reply:
Is it a new test case? Why was there no sleep before?

Reply:
Will remove it, i guess this is not necessary

Reply:
Removed